SPECS/SymCrypt/SymCrypt.spec

```diff
@@ -1,15 +1,17 @@
-%define debug_package %{nil}
 Summary:        A core cryptographic library written by Microsoft
 Name:           SymCrypt
 Version:        103.4.2
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System/Libraries
 URL:            https://github.com/microsoft/SymCrypt
 Source0:        https://github.com/microsoft/SymCrypt/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Source1:        https://github.com/smuellerDD/jitterentropy-library/archive/v3.3.1.tar.gz#/jitterentropy-library-3.3.1.tar.gz
+Source2:        find-debuginfo
+Patch1:         0001-add-build-flags-to-prevent-stripping-and-post-proces.patch
+Patch2:         0001-add-parameter-to-process_fips_module-to-specify-the-.patch
 BuildRequires:  cmake
```

Comment:
Nit-pick: could you only modify the patch names a bit, remove the noise like "0001" or make them concise enough, so that they are not truncated like the "the-" ending of the second one? Nothing big, just so it's easier for the reader.

Reply:
I at least mostly agree. If I end up making a different change before this is merged, I'll change them. If not, it's not the end of the world: These changes have been accepted upstream so at some point we'll get rid of these patches altogether.